app/src/androidTest/java/com/sdp/movemeet/view/home/HomeScreenActivityLoggedTest.java

```diff
@@ -72,10 +72,4 @@ public void noAccountHasCorrectIntentWhenLogged() throws InterruptedException {
         onView(withId(R.id.noAccountButton)).perform(click());
         intended(hasComponent(MainActivity.class.getName()));
     }
-
```

Comment:
So now the record button is in the activity description ?

Reply:
Yes, since recordings are always tied to an activity, there is no need for a record button on the home screen which could not save the data with an activity

Reply:
Yes, and this is also important to fulfil the "sensor requirement" by fully integrating the GPS tracking feature into the core of our app (and not having it "simply" on the HomeScreen).